### PR TITLE
More words

### PR DESCRIPTION
**Pull request by sebweb3r**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -16564,7 +16574,8 @@ lpatform->platform
 lsat->last
 lsit->list
 lsits->lists
-lukid->likud
+lukid->lucid, likud,
```
Comment (sebweb3r) [inline codespell_lib/data/dictionary.txt:16577]: likud is the name of an israelian party. should it be a suggestion?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -10068,6 +10070,9 @@ ecret->secret, erect,
 ecspecially->especially
 ect->etc
 ecxept->except
+ecxite->excite
+ecxited->excited
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:10074]: -ing ?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -11437,6 +11442,9 @@ existsing->existing
 existung->existing
 existy->exist
 existying->existing
+exitation->excitation
+exitations->excitations
+exite->exit, excite,
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:11447]: also exits ?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -15899,6 +15911,7 @@ iresistibly->irresistibly
 iritable->irritable
 iritated->irritated
 ironicly->ironically
+irradation->irradiation
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:15914]: also irradate->irradiate ?

(and perhaps also iradiate, iradate, iradiation, iradation etc.)

Could maybe also add `irigation->irrigation` (etc.) if you're feeling enthusiastic? :wink: 

And I've just spotted (sorry!) that the dictionary has `iritated->irritated` but not `iritate->irritate`.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -16564,7 +16577,8 @@ lpatform->platform
 lsat->last
 lsit->list
 lsits->lists
-lukid->likud
+lukid->lucid, likud,
+lumination->lamination, illumination,
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:16581]: Hmmm? https://www.collinsdictionary.com/dictionary/english/lumination

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -16564,7 +16577,8 @@ lpatform->platform
 lsat->last
 lsit->list
 lsits->lists
-lukid->likud
+lukid->lucid, likud,
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:16580]: Maybe also add `lusid->lucid` ?

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 7]
```diff
@@ -101,6 +101,8 @@ organisations->organizations
 organise->organize
 organised->organized
 plagiarise->plagiarize
+polarise->polarize
+polarised->polarized
```
Comment (lurch) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:105]: Also the valid ones from this list? http://app.aspell.net/lookup?dict=en_GB-ise&words=polarise%0D%0Apolariser%0D%0Apolarising%0D%0Apolarised%0D%0Apolarises

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -14271,6 +14279,10 @@ illustrasion->illustration
 illution->illusion
 ilness->illness
 ilogical->illogical
+iluminate->illuminate
+iluminated->illuminated
+iluminates->illuminates
+ilumination->illumination
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:14285]: illuminations?

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:16581]: All Dictionaries I have looked it up, don't know it.
https://dictionary.cambridge.org/de/spellcheck/englisch/?q=lumination

I will have a look on some physics papers later, if it was/is used by people.

Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:16581]: I think most people, who use it, want something like luminance?!

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:15914]: added stuff, except irigat*

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:16585]: Only use of lusid look intentional 

https://www.grep.app/search?q=lusid&words=true

Comment (lurch) [reply codespell_lib/data/dictionary.txt:16585]: Fair enough, I was just thinking that somebody who is bad at spelling, might misspell it that way. But perhaps I was over-reaching.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:16585]: Yeah probably. The fact someone added that as the initial entry hopefully means they found it as a typo. I think putting it second is right though. Should it be capitalised?

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:16586]: Chuck it in rare:
http://app.aspell.net/lookup?dict=en_US&words=lumination%0D%0Aillumination

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -20874,6 +20893,7 @@ programable->programmable
 programatic->programmatic
 programatically->programmatically
 programattically->programmatically
+programed->programmed
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:20896]: You could also add `programmd->programmed`.

Also not yours but I've just seen `programm->program` should also suggest the BE `programme`

**Review by peternewman — CHANGES_REQUESTED** [review]
Plus the capitalisation of Likud if necessary

File: codespell_lib/data/dictionary.txt (hunk 9 above)
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:20896]: I wanted to add programme->program to enGB_to_enUS, 
but it is not in aspell http://app.aspell.net/lookup?dict=en_US&words=programm%0D%0Aprogramme

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -20874,12 +20892,15 @@ programable->programmable
...
+programd->programmed
+programed->programmed
 programemer->programmer
 programemers->programmers
 programers->programmers
 programing->programming
-programm->program
+programm->program, programme,
 programmaticaly->programmatically
+programmd->programmed
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:20903]: I think this should suggest programme too given the d/e proximity.

**Review by peternewman — CHANGES_REQUESTED** [review]
Just one comment from me

File: codespell_lib/data/dictionary.txt (hunk 10 above)
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt]: added and rebased.

**Review by peternewman — APPROVED** [review]

File: codespell_lib/data/dictionary.txt (hunk 9 above)
Comment (lurch) [reply codespell_lib/data/dictionary.txt:20913]: It is if you look in the right dictionary? http://app.aspell.net/lookup?dict=en_GB-ise&words=program%0D%0Aprogramme

The way that I've always understood it, is that in the UK we have television programmes and computer programs, but in the US they use 'program' for both.

Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:20913]: Ok, I relied on the feature "show it to me, if it is in other dictionaries." which doesn't seem to work here. My local aspell test fails too.
let's try, what jenkins says to #1688

**Pull request merged**